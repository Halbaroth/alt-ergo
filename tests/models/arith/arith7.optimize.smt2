(set-option :produce-models true)
(set-logic ALL)
(declare-const x Real)
(push 1)
(assert (< x 0.5))
(maximize x)
(check-sat)
(get-model)
(get-objectives)
(pop 1)
(push 1)
(assert (< x (- 0.0 0.5)))
(maximize x)
(check-sat)
(get-model)
(get-objectives)
(pop 1)
