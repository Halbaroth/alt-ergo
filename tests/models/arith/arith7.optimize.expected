
unknown
; This model is a best-effort. It includes symbols
; for which model generation is known to be incomplete.
(
  (define-fun x () Real (/ 7 20))
  (define-fun y () Real 1.0)
)
(objectives 
  (x (- (/ 1 2) epsilon))
  (y (+ oo))
)